engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -537,18 +537,18 @@ public float getBlockLightIntensityAt(Vector3f position) {
 
     @Override
     public String getMetrics() {
-        String stringToReturn = "";
-        stringToReturn += renderableWorld.getMetrics();
-        stringToReturn += "Empty Mesh Chunks: ";
-        stringToReturn += statChunkMeshEmpty;
-        stringToReturn += "\n";
-        stringToReturn += "Unready Chunks: ";
-        stringToReturn += statChunkNotReady;
-        stringToReturn += "\n";
-        stringToReturn += "Rendered Triangles: ";
-        stringToReturn += statRenderedTriangles;
-        stringToReturn += "\n";
-        return stringToReturn;
+        StringBuilder stringToReturn = new StringBuilder("");
```

Comment:
No, I'm using Intellij... I was using a rule of thumb that if you have to concatenate many variables into a single string, a builder is more efficient.